laurent22/joplin
# Mobile: Resolves #10592: Make mobile plugin settings screen UI closer to desktop

# Summary

This pull request makes the plugin screen UI on mobile closer to the desktop UI.

Resolves #10592.


# Screenshots

**Searching, installed plugin in search results**:
![screenshot](https://github.com/laurent22/joplin/assets/46334387/45e1736c-495b-489f-b7dc-d2cc65ff4872)

**Screen when there is no search query**:
![screenshot](https://github.com/laurent22/joplin/assets/46334387/f0c4926d-873b-4891-9999-1b2fd688a393)

**Searching, large screen, incompatible and updatable plugins in search results**:
![screenshot](https://github.com/laurent22/joplin/assets/46334387/35dbaef1-2dfd-4afe-bb3b-4920ef326fa1)

**When no plugins are installed, "No plugins are installed." is shown below the search input**:
![image](https://github.com/laurent22/joplin/assets/46334387/dd6969b4-c9df-4644-b30a-fcf445f1456a)


# Testing plan

1. Search for ` `.
2. Tap on the recommended badge for a plugin. Verify that it shows an alert describing recommended plugins.
3. Install an outdated version of a plugin present in the store.
4. Verify that the "Update available" badge is visible.
5. Update the plugin.
6. Uninstall all plugins.
7. Verify that "No plugins are installed" is shown below the search input.

This has been tested successfully on Android 13.
